src/ontology/cl-edit.owl

```diff
@@ -31844,14 +31845,16 @@ EquivalentClasses(obo:CL_4040004 ObjectIntersectionOf(obo:CL_0002570 ObjectSomeV
 
 # Class: obo:CL_4040005 (mesenchymal stem cell of apical papilla)
 
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:30834270") Annotation(oboInOwl:hasDbXref "PMID:34026757") obo:IAO_0000115 obo:CL_4040005 "A mesenchymal stem cell part of the apical papilla tooth root.")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:30834270") Annotation(oboInOwl:hasDbXref "PMID:34026757") obo:IAO_0000115 obo:CL_4040005 "A mesenchymal stem cell that is part of the apical papilla tooth root.")
 AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4040005 <https://orcid.org/0000-0002-0819-0473>)
 AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4040005 "2023-11-02T09:44:47Z"^^xsd:dateTime)
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:30834270") Annotation(oboInOwl:hasSynonymType obo:OMO_0003000) oboInOwl:hasExactSynonym obo:CL_4040005 "SCAPs")
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:30834270") oboInOwl:hasExactSynonym obo:CL_4040005 "stem cells from the apical papilla")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:30834270") Annotation(oboInOwl:hasSynonymType obo:OMO_0003004) oboInOwl:hasExactSynonym obo:CL_4040005 "stem cells from the apical papilla")
```

Comment:
Since stem cell is broader than mesenchymal stem cell, I am thinking that it would be more adequate to be a broad synonym. Everything else looks good to me, so once the edit is done I will accept the PR.

Reply:
OK